ADR/0033-upstreaming-integration-service.md

```diff
@@ -0,0 +1,228 @@
+# XX. Upstreaming Integration Service
+
+Date: 2023-11-05
+
+## Status
+
+Proposed
+
+## Authors
+
+- Ralph Bean
+
+## Context
+
+This ADR is part of series aiming to address *independent* usage of our controllers and services to
+see if they are viable on their own. We have always assumed they would be used together, and
+therefore some architectural assumptions need to be identified and changed in order to decouple
+them.
+
+See also [Decoupling Deployment] for related context.
+
+Today, [integration-service] does the following things:
+
+- **Snapshot Construction**: When a pipelines-as-code (PaC) PipelineRun completes,
+  [integration-service] uses the [Component] API (the "global candidate list") to construct
+  a [Snapshot], in order to trigger testing.
```

Comment:
I think that this could be expanded because Snapshot creation is primarily there to create a version of the Application which can then be tested, released and deployed.

Reply:
Hm, ok. After reading the rest of the ADR do you think it's still helpful to add that context here?

Reply:
You're right, it's ok as-is.